pynsee/utils/init_conn.py

```diff
@@ -74,7 +77,7 @@ def init_conn(insee_key, insee_secret, http_proxy="", https_proxy=""):
             "!!! Token is missing, please check insee_key and insee_secret are correct !!!"
         )
     else:
-        print(f"Token has been created")
+        logger.debug(f"Token has been created")

```

Comment:
move to info

Reply:
I'm not sure you need to "info" that : it the token generation fails, you get an exception anyway, so that seems obvious to me.

Reply:
the way I see it, a non-developer may want to check that all went well (e.g. to confirm an error in their script does not come from there), whereas debug is strictly limited to things that require an understanding of the library's internals and that would be used to debug of the library itself (and not just of a script using the library)

Reply:
it should be printed as an info message